### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by GhostEsso**
- I sent and received data from an API.
- I used API documentation.
- I understood and used JSON.
- I made JavaScript code asynchronous.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -96,23 +128,98 @@ class LeaderboardApp {
...
+        li.style.backgroundColor = index % 2 === 0 ? 'white' : 'silver'; // Alternate white and silver backgrounds
+        ul.appendChild(li);
+      });
+    } catch (error) {
+      throw new Error('Failed to update scores:', error);
+    }
+  }
+
+  // Modify createScoreForm method to call submitScore on form submission
+  createScoreForm(gameId) {
```
Comment (usorfaitheloho) [inline src/index.js]: You did well with your project, kindly note according to the requirement, the `refresh` button should fetch the recently updated scores but this doesn't happen in your project and instead it fetches everything in the database. Please this is an abnormal behaviour that should be corrected, kindly ensure that the `refresh` button fetches only the recently inputted values 😄 

Project Requirement :
image


Your project :

https://github.com/GhostEsso/Leaderboard/assets/20537712/ee4bc5a8-4593-4559-88f2-b71c07b4fd0c

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @GhostEsso  ,
> 
> Great job for implementing  some of the Project Requirement, but there are some changes that need to be implemented in order to proceed to the next project 👍
> 
> ## Highlights 👏🏻
> ✔️ No linters error ✔️ You used correct Git Hub-Flow. ✔️ Good README ✔️ Good Commit Messages
> 
> almost 

> ## Required Changes ♻️
> _Check the comments below the review._
> 
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (GhostEsso) [reply src/index.js]: @usorfaitheloho Thanks, I understand. I will correct this. :+1:

**Review by julie-ify — APPROVED** [review]
Hi @GhostEsso,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GhostEsso** [comment]
@julie-ify @usorfaitheloho Thank you for reviewing my code! :pray:

**Pull request merged**